src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/ICounterPayload.cs

```diff
@@ -41,6 +41,12 @@ internal interface ICounterPayload
         /// </summary>
         string Metadata { get; }
 
-        EventType EventType { get; set; }
+        EventType EventType { get; }
+
+        bool IsMeter { get; }
+
+        int Series { get; }
+
+        bool IsValuePublishedEvent { get => EventType != EventType.BeginInstrumentReporting; }
```

Comment:
I'd suggest enumerating the event types which do qualify rather than those which do not, it makes the intent more clear. The current impl is also incorrectly triaging a bunch of events. For example error payloads shouldn't return true I think.

For consistency I'd add this to the EventType extensions so its less likely to be missed if new events are added.

Reply:
Sorry, I misinterpreted the initial ask - since the exceptions were already being filtered out earlier in this conditional I didn't account for them. If I should have included any other `EventTypes`, just let me know and I'll add them in.